src/app/core/admin/admin-menu/admin-menu-item/admin-menu-item.component.html

```diff
@@ -0,0 +1,48 @@
+<h2 mat-dialog-title>
+  {{ !isNew ? 'Edit Menu Item' : 'Adding New Menu Item' }}
```

Comment:
To make things easier to translate, it's good to not have any conditions inside an i18n element
```suggestion
  @if(isNew) {
    <ng-container i18n>Add New Menu Item</ng-container>
  } @else {
    <ng-container i18n>Edit Menu Item</ng-container>
  }
```